chat/chat-impl/impl/src/java/org/sakaiproject/chat2/model/impl/ChatEntityProducer.java

```diff
@@ -179,7 +177,8 @@ public String archive(String siteId, Document doc, Stack stack, String archivePa
       }
       catch (Exception any)
       {
-         log.warn("archive: exception archiving service: " + serviceName());
+         log.error("exception archiving site " + siteId + ": " + serviceName(), any);
```

Comment:
warn was the correct level as the request can continue processing.
```suggestion
         log.warn("Failed archiving chat data for site {}:{}, {}", siteId, serviceName(), any.toString());
```